### ISSUE TITLE
Errors when compiling

### ISSUE THREAD

**Issue by Miss-Inputs** [perilsensitive/cxnes#2]
When cloning from git and compiling, I get the following error from make:
`make: *** No rule to make target '.git/refs/tags/*', needed by 'main/version.c'.  Stop.`

If I edit the makefile to remove the reference to that directory in VERSION_C_DEPS, so that line looks like this:
`VERSION_C_DEPS = $(shell git rev-parse --is-inside-work-tree > /dev/null 2>&1 && echo .git/HEAD .git/packed-refs .git/refs/heads/*) VERSION`
it seems to compile fine, but I don't know if that's a sign I did something terribly wrong as it seems to be an empty directory.

Also when doing a sudo make install I get this error:
```
make[1]: Entering directory '/tmp/cxnes'
 /usr/bin/mkdir -p '/usr/local/bin'
  /usr/bin/install -c cxnes '/usr/local/bin'
/usr/bin/mkdir -p /usr/local/share/cxnes/cheatdb
/usr/bin/install -c -m 644 data/cheatdb/* /usr/local/share/cxnes/cheatdb
/usr/bin/install: cannot stat 'data/cheatdb/*': No such file or directory
make[1]: *** [Makefile:2335: install-data-local] Error 1
make[1]: Leaving directory '/tmp/cxnes'
make: *** [Makefile:1797: install-am] Error 2
```

If I comment out the line in the makefile for the install-data-local target, it'll continue as normal.

Otherwise the emulator works great after I installed it. Yay!

**Comment by perilsensitive** [comment]
Yeah, sorry about the build issues.  I haven't had time to work on them yet; they work on my box but may not work on anyone else's.  I'll try to clean them up in the next couple of days.

**Comment by perilsensitive** [comment]
Commited a change to resolve the cheatdb error; haven't addressed the other one yet.
